membership-attribute-service/app/services/ScanamoAttributeService.scala

```diff
@@ -8,14 +8,16 @@ import com.gu.scanamo.syntax.{set => scanamoSet, _}
 import com.gu.scanamo.update.UpdateExpression
 import com.typesafe.scalalogging.LazyLogging
 import models.Attributes
+import monitoring.Metrics
 import org.joda.time.{DateTime, LocalDate}
 import play.api.libs.concurrent.Execution.Implicits._
-
 import scala.concurrent.Future
 
 class ScanamoAttributeService(client: AmazonDynamoDBAsync, table: String)
     extends AttributeService with LazyLogging {
 
+  val metrics = Metrics("ScanamoAttributeService") //referenced in CloudFormation
```

Comment:
Having to copy the string between here and Cloudformation doesn't feel like a great solution (despite this comment, which tries to warn people, I feel like this could get broken quite easily). Does anyone have a better idea?

Reply:
How about tagging the EC2 instance in the cloudformation, use the AWS API to read the tag from the local instance? But then you have the same problem with the tag name!

Reply:
Hmm interesting suggestion, I think I'll go with this approach for now and re-visit if we can think of something cleaner...